cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -203,6 +204,34 @@ def scale_ingress_controller():
         hookenv.log('Failed to scale ingress controllers. Will attempt again next update.')  # noqa
 
 
+@when('config.changed.labels', 'kubernetes-worker.config.created')
+def apply_node_labels():
```

Comment:
Will this run on initial deployment? Is the `config.changed.labels` state true in that case?